demos/node-classification/gcn/gcn-cora-node-classification-example.ipynb

```diff
@@ -509,7 +509,7 @@
     }
    ],
    "source": [
-    "x_inp, x_out = gcn.build()\n",
+    "x_inp, x_out = gcn.in_out_tensors()\n",
```

Comment:
There's text above that should be updated ``via the `GCN.build` method:`` -> ``via the `GCN.in_out_tensors` method:``.

Reply:
nice catch! I found another one in the cluster gcn demo